airflow/providers/google/cloud/hooks/dataproc.py

```diff
@@ -985,6 +985,68 @@ def list_batches(
         )
         return result
 
+    @GoogleBaseHook.fallback_to_default_project_id
+    def wait_for_batch(
+        self,
+        batch_id: str,
+        region: str,
+        project_id: str,
+        wait_time_interval: int = 10,
```

Comment:
does it make sense to add this param in the operator and keep the default value of 10 sec, this way a dag author can decide how long want to wait otherwise currently it 10 sec always, WDYT?

Reply:
It does. I moved it up to the operator constructor.